bpf/tests/conntrack_test.c

```diff
@@ -215,4 +215,85 @@ int bpf_test(__maybe_unused struct __sk_buff *sctx)
 	test_finish();
 }
 
+CHECK("tc", "conntrack_svc")
+int svc_test(__maybe_unused struct __sk_buff *sctx)
+{
+	test_init();
+
+	TEST("ct_lookup_svc", {
+		struct __ctx_buff ctx = {};
+		int res;
+		struct ipv4_ct_tuple tuple = {};
+		struct ct_state ct_state = {};
+		union tcp_flags seen_flags = {0};
+		__u32 monitor;
+
+		tuple.nexthdr = IPPROTO_TCP;
+		tuple.flags = CT_SERVICE;
+
+		struct ct_entry ct_entry_new = {};
+
+		res = map_update_elem(get_ct_map4(&tuple), &tuple, &ct_entry_new, BPF_ANY);
+		if (IS_ERR(res))
+			test_fatal("map_update_elem: %lld", res);
+
+		struct ct_entry *entry = map_lookup_elem(get_ct_map4(&tuple), &tuple);
+
+		if (!entry)
+			test_fatal("ct entry lookup failed");
+
+		seen_flags.value |= TCP_FLAG_SYN;
+
+		/* First packet is monitored */
+		res = __ct_lookup(get_ct_map4(&tuple), &ctx, &tuple,
+				  ct_tcp_select_action(seen_flags), CT_SERVICE,
+				  CT_ENTRY_SVC, &ct_state, true, seen_flags, &monitor);
```

Comment:
CI noticed that the function signature in v1.15 is different. I can queue up a manual backport if you prefer, haven't checked *how* big the difference is.

Reply:
`s/CT_ENTRY_SVC/CT_ENTRY_ANY` should suffice :)

Reply:
Thanks, will apply!